core/node/eth_sender/src/eth_tx_aggregator.rs

```diff
@@ -268,12 +287,31 @@ impl EthTxAggregator {
                 )));
             }
             let default_aa = H256::from_slice(&multicall3_default_aa);
+
+            let evm_simulator = if evm_simulator_hash_requested {
+                let multicall3_evm_simulator =
+                    Multicall3Result::from_token(call_results_iterator.next().unwrap())?
+                        .return_data;
+                if multicall3_evm_simulator.len() != 32 {
+                    return Err(EthSenderError::Parse(Web3ContractError::InvalidOutputType(
+                        format!(
+                            "multicall3 evm simulator hash data is not of the len of 32: {:?}",
+                            multicall3_evm_simulator
+                        ),
+                    )));
+                }
+                Some(H256::from_slice(&multicall3_evm_simulator))
+            } else {
+                None
+            };
+
             let base_system_contracts_hashes = BaseSystemContractsHashes {
                 bootloader,
                 default_aa,
+                evm_simulator,
             };
 
-            call_results_iterator.next().unwrap();
+            call_results_iterator.next().unwrap(); // FIXME: why is this value requested?
```

Comment:
It looks like this value (`getVerifierParams` output) is always unused.

Reply:
leftover, `getVerifierParams` call can be removed